### ISSUE TITLE
cloudscheduler defaults set <server> doesn't set default server

### ISSUE THREAD

**Issue by rseuster** [hep-gc/cloudscheduler#120]
on csv2-dev:
```
cloudscheduler defaults set -s dev -sa https://csv2-dev.heprc.uvic.ca -su seuster -spw <passwd>
```
then
```
cloudscheduler cloud list
Error: No server specified and no server address (URL) provided.
```
One needs to do an additional
```
cloudscheduler cloud list -s dev
```
to set the default.

**Comment by crlb** [comment]
fixed dd777dab9de0ede412eca5e5f21caa001367e7d9

**Comment by rseuster** [comment]
closing
